pkg/storage/minio/minio.go

```diff
@@ -53,3 +54,10 @@ func NewStorage(conf *config.MinioConfig, timeout time.Duration) (storage.Backen
 	}
 	return &storageImpl{minioClient, minioCore, bucketName}, nil
 }
+
+// TrimHTTP trims "http://" or "https://" prefix from input string.
```

Comment:
Can you expand on this comment, maybe even linking to the issue would help with context.

Reply:
```suggestion
// TrimHTTP trims "http://" or "https://" prefix from input string.
// By default, Minio doesn't need to specify protocol in the URL so it should be trimmed. 
// Related issue: https://github.com/gomods/athens/issues/1938#issuecomment-2067590653
```

Reply:
Is this okay?

Reply:
That looks great!

Reply:
Thanks